cmd/snap/cmd_snapshot.go

```diff
@@ -43,10 +43,67 @@ type savedCmd struct {
 	} `positional-args:"yes"`
 }
 
-var shortSavedHelp = i18n.G("List currently stored snapshots")
+var (
+	shortSavedHelp   = i18n.G("List currently stored snapshots")
+	shortSaveHelp    = i18n.G("Save a snapshot of the current data")
+	shortForgetHelp  = i18n.G("Delete a snapshot")
+	shortCheckHelp   = i18n.G("Check a snapshot")
+	shortRestoreHelp = i18n.G("Restore a snapshot")
+)
+
 var longSavedHelp = i18n.G(`
-The saved command lists the snapshots that have been created previously with
-the 'save' command.
+The saved command lists the snapshots that have been created
```

Comment:
@chipaca could we do some ANSI art magic here to apply bold or italics to the actual command being mentioned. It would show for a good chunk of people and would immediately look better IMO (though not in this PR)

Reply:
I've been wanting to find a way to do this for a while. What I want to do is have a formatter that takes e.g. `%Q` to mean "special-quoted" or something, that uses italics and typographic quotes if available, falling back to simple quotes.

It's fiddly work, and would mess up widths if done without care. I know how to do it, but I haven't gotten to it yet.